packages/web-client/app/components/card-pay/layer-two-connect-card/index.hbs

```diff
@@ -47,8 +47,8 @@
         {{!-- future: show Prepaid Cards, Badges --}}
         <CardPay::LabeledValue
           @label="Status"
-          @value="Connected"
-          @icon="success-bordered"
+          @value={{if this.isConnected "Connected" "Disconnected"}}
+          @icon={{if this.isConnected "success-bordered" "failure-bordered"}}
```

Comment:
While the red icon could easily be just a couple of lines of CSS, I added a new SVG icon for the sake of consistency. What are your thoughts on this?

Reply:
New SVG icon sounds better than constructing a circle with CSS! CSS circles can be a bit fragile.